R/fit.R

```diff
@@ -0,0 +1,78 @@
+#' Internal objective function for least-squares non-linear fitting of target usage
+#'
+#' @param distribution A vector of proposed distribution
+#' @inheritParams fit_usage
+#'
+#' @return Sum of squared difference between proposed usage and target usage
+fit_usage_objective <- function(distribution, distribution_timesteps,
+                                target_usage, target_usage_timesteps,
+                                population, net_loss, timesteps, rho,
+                                seed){
+  set.seed(seed)
+  pu <- population_usage(
+    net_loss = net_loss,
+    population = population,
+    timesteps = timesteps,
+    distribution = distribution,
+    distribution_timesteps =  distribution_timesteps,
+    rho = rho)
```

Comment:
What is the correlation between rounds? The correlation between who gets nets per round?

Reply:
👍 yep